content/en/docs/concepts/cluster-administration/monitoring.md

```diff
@@ -0,0 +1,40 @@
+---
+reviewers:
+- dashpole
+title: Monitoring Architecture
+content_template: templates/concept
+weight: 60
+---
+
+{{% capture overview %}}
+
+TODO Overview
```

Comment:
Can you fix all of the TODO's on this page or remove the doc entirely?

Reply:
whoops, that wasn't meant to be included.  It is removed